src/rebuild/scan.c

```diff
@@ -521,7 +521,7 @@ rebuild_scanner(void *data)
 	/* Create object tree root */
 	memset(&uma, 0, sizeof(uma));
 	uma.uma_id = UMEM_CLASS_VMEM;
-	rc = dbtree_create(DBTREE_CLASS_NV, 0, 4, &uma, NULL,
+	rc = dbtree_create(DBTREE_CLASS_UV, 0, 4, &uma, NULL,
```

Comment:
This only addresses container tree, usually there are only few containers in testing, so I think it's not the hash collision source. (actually, I tried this)

I think the problem could be that DBTREE_CLASS_NV is used by object tree during rebuild. (As I mentioned, the hash collision always happens when I increase object count from 5000 to 20000).